src/plonetheme/tokyo/browser/overrides/plone_app_z3cform/plone.app.z3cform.templates.macros.pt

```diff
@@ -105,24 +114,21 @@
             </tal:block>
 
           </metal:define>
-        </metal:fields-slot>
 
-        <metal:block define-slot="belowfields" />
```

Comment:
Not sure what this was implemented for... Are you sure we don't need it anymore?